File: Scripts/laptop/classes/r/classes.r
executeTask <- function(i) {
	
myClassVariable <- list(Variable="193578")
class(myClassVariable) <- append(class(myClassVariable), "Value")

GetVariable <- function(x) {
	UseMethod("GetVariable",x,i)
}

GetVariable.Value <-function(x,i) {
	return(x$Variable + i)
}
	return(i)
}

r = 1

for(i in 0:2100000000) {
	r = executeTask(i + r)
}

File: Scripts/laptop/classes/c-sharp/classes.cs
using System;

class MyClass{

  // instance variable
  private volatile int variable;  // Note: instance variables are usually "private"

  /**
  * The constructor
  */
  public MyClass(){
    // creates a new instance
  }

  /**
  * A method
  */
  public void someMethod(){
   this.variable = 1;
  }

  private static volatile int r = 1;

  public static int executeTask(int i) {

	MyClass obj = new MyClass();
	obj.someMethod();

	return i + 1;
  }

public static void Main(String[] args){

	for (int i = 0; i < 2000000000; ++i) {
		r = executeTask(i + r);
	}
}

}

